### PR TITLE
Config with default template

### PR DESCRIPTION
**Pull request by gforcada**

### CODE REVIEW

**Comment by gforcada** [comment]
@jenkins-plone-org please run jobs

**Review by mauritsvanrees — CHANGES_REQUESTED** [review]
LGTM, except I think we should keep the `plone.app.vocabularies` dependency.  See inline comment.

File: setup.py [hunk 1]
```diff
@@ -1,67 +1,68 @@
...
-    'setuptools',
-    'zope.component',
-    'zope.contentprovider',
-    'zope.interface',
-    'zope.site',
-    'zope.viewlet',
+    "Acquisition",
+    "Products.GenericSetup",
+    "ZODB",
+    "Zope",
```
Comment (jensens) [inline setup.py:24]: shouldnt Zope cover Acquisition, zope.component, zope.interface and ZODB at least?

File: setup.py [hunk 2]
```diff
@@ -22,12 +22,11 @@
     "Products.GenericSetup",
     "ZODB",
     "Zope",
-    "plone.app.vocabularies",
```
[the remark this answers is not in the document]
Comment (gforcada) [reply setup.py]: Indeed, let me fix that 👍🏾

File: setup.py (hunk 1 above)
Comment (gforcada) [reply setup.py:22]: indeed, fixed!

**Review by mauritsvanrees — APPROVED** [review]

**Comment by mauritsvanrees** [comment]
Thanks for the fixes.  Let's check one more time:

@jenkins-plone-org please run jobs

**Review by jensens — APPROVED** [review]

**Pull request merged**